cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html

```diff
@@ -49,53 +51,58 @@
                 {% endif %}
             {% endif %}
         </div>
-        <div class="m-image-text-25-75_content">
-            <h3 class="m-image-text-25-75_heading">{{ value.heading }}</h3>
-
-            <p class="m-image-text-25-75_body">{{ value.body | safe }}</p>
 
+        <div class="m-image-text-25-75_content">
+            <h3>{{ value.heading }}</h3>
+            <p>{{ value.body | safe }}</p>
```

Comment:
I am not really sure, it's the same as before, I just removed the extra line break between them while cleaning things up a bit.